### PR TITLE
[JBPM-10004] GET "instances/{processInstanceId}/timers" endpoint is not returning process SLA timers

### PR DESCRIPTION
**Pull request by elguardian**
Jira: https://issues.redhat.com/browse/JBPM-10004

### CODE REVIEW

**Review by gmunozfe — APPROVED** [review]
Looks good to me, works also with timer expressions.
Nailed down @elguardian !

**Comment by gmunozfe** [comment]
Tested with integration worked fine as well. The only point (enhancement) here is to add a name, same to SLA for tasks, with [SLA]process.id, wdyt @elguardian ?

File: jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/admin/commands/ListTimersCommand.java [hunk 1]
```diff
@@ -66,6 +66,12 @@ public ListTimersCommand(long processInstanceId) {
         	throw new ProcessInstanceNotFoundException("No process instance can be found for id " + processInstanceId);
         }
 
+        Long processSlaTimer = wfp.getSlaTimerId();
+        if (processSlaTimer != null && processSlaTimer != -1L) {
+            timers.add(buildTimer(tm.getTimerMap().get(processSlaTimer)));
```
Comment (gmunozfe) [inline jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/admin/commands/ListTimersCommand.java:71]:
```suggestion
           TimerInstanceImpl details = buildTimer(tm.getTimerMap().get(processSlaTimer));
            details.setTimerName("[SLA-Process]" + wfp.getProcessName());
            timers.add(details);
```

Comment (elguardian) [reply jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/admin/commands/ListTimersCommand.java:71]: done

**Pull request merged**